ledger/src/shred/merkle.rs

```diff
@@ -1479,6 +1452,17 @@ mod test {
         }
     }
 
+    // Maps number of (code + data) shreds to merkle_proof.len().
+    fn get_proof_size(num_shreds: usize) -> u8 {
```

Comment:
Maybe this belongs in merkle_tree.rs so people might find it easier if needed for things other then tests?

Reply:
yeah, that feels like a better spot... but we should probably move the tests over too. I'm going to punt this to a follow-up PR if that's cool